dao/src/main/java/org/thingsboard/server/dao/attributes/CachedAttributesService.java

```diff
@@ -234,7 +234,7 @@ public ListenableFuture<String> save(TenantId tenantId, EntityId entityId, Strin
         validate(entityId, scope);
         AttributeUtils.validate(attribute, valueNoXssValidation);
         ListenableFuture<String> future = attributesDao.save(tenantId, entityId, scope, attribute);
-        return Futures.transform(future, key -> evict(entityId, scope, attribute, key), cacheExecutor);
+        return Futures.transform(future, key -> put(entityId, scope, attribute, key), cacheExecutor);
```

Comment:
I strongly suggest to write the test that will execute multiple threads with updates of the same attribute. You may cache the wrong value since it is processed asynchronously.